src/components/table/nz-table.component.ts

```diff
@@ -220,7 +225,7 @@ export class NzTableComponent implements AfterViewInit, OnInit {
   ngOnInit() {
     const scrollbarWidth = measureScrollbar();
```

Comment:
This line includes `measureScrollbar` function are unnecessarily according to your modifications

Reply:
Ok, thank